els-back/app/Http/Controllers/User/AuthenticationController.php

```diff
@@ -58,12 +58,19 @@ public function logout(Request $request)
         return "success";
     }
 
+    public function getActivities($followed_ids) 
```

Comment:
i think user_ids is more appropriate since we can use this function not only for followed users.
```suggestion
    public function getActivities($user_ids) 
```

Reply:
This was changed in the feature/users branch sir, I tried to rebase multiple times from the develop branch(up to date with feature/users) but it does not retrieve from other branches